pages/ksphere/kommander/1.0/clusters/attach-eks-cluster/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: layout.pug
+navigationTitle: Attach Amazon EKS Cluster to Kommander
+title: Attach Amazon EKS Cluster to Kommander
+menuWeight: 7
+excerpt: Attach an existing EKS cluster to Kommander
+beta: true 
+enterprise: false
+---
+
+You can attach any existing Kubernetes clusters to Kommander, then use it to examine and manage those clusters later. The following procedure shows how to attach an existing Amazon Elastic Kubernetes Service (EKS) cluster to Kommander. 
+
+## Before you begin 
+
+This procedure requires the following items and configurations:
+
+- A fully configured and running Amazon [EKS](https://aws.amazon.com/eks/) cluster.
+- Konvoy v1.5.0-beta.0 or above, [installed and configured](https://docs.d2iq.com/ksphere/konvoy/1.5.0-beta/install/) for your Amazon EKS cluster, on your machine.
+- Kommander v1.0.1-beta.1 or above, [installed and configured](https://docs.d2iq.com/ksphere/kommander/1.1.0-beta/install/) on your machine.
+
+<p class="message--note"><strong>NOTE: </strong>This procedure assumes you have at least one existing and spun up Amazon EKS cluster. Refer to the Amazon EKS [website](https://aws.amazon.com/eks/) for setup and configuration information.</p>
+
+## Attach Amazon EKS Clusters to Kommander
+
+1. Ensure you are connected to your first EKS cluster. Enter the following commands:
+
+    ```bash
+    kubectl config get-contexts
+    kubectl config use-context <context for first eks cluster>
+    ```
+
+1. Confirm `kubectl` can access the EKS cluster.  
+
+    ```bash
+    kubectl get no
+    ```
+    
+1. Create a service account for Kommander on your EKS cluster.
+
+    ```bash
+    kubectl -n kube-system create serviceaccount kommander-cluster-admin
+    ```
+    
+1. Configure your `kommander-cluster-admin` service account to have `cluster-admin` permissions. Enter the following commands:
+
+    ```yaml
+    cat << EOF | kubectl apply -f -
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRoleBinding
+    metadata:
+      name: kommander-cluster-admin
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: cluster-admin
+    subjects:
+    - kind: ServiceAccount
+      name: kommander-cluster-admin
+      namespace: kube-system
+    EOF
+    ```
+
+1. Create a kubeconfig file compatible with the Kommander UI. Enter these commands to set the following environment variables:
```

Comment:
how do they know if it's compatible with the Kommander UI?

Reply:
They really don't but this set of commands DOES create a compatible kubeconfig file.

Reply:
correct. this whole rigamarole is due to the fact that the default kubeconfig file used/created by EKS is not compatible with the kommander UI.

Reply:
Is there already a place where we describe how to create this file in the docs? If so, link those words to that topic. If not, we need to add a JIRA to document this rigamarole. :)

Reply:
ohhh! I see @russurquhart1 comment now. I'd probably reword to: 

Create a kubeconfig file according to the instructions below, which will make the file compatible with Kommander.